spec/integration/posts_show_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe 'Posts show page', type: :system do
   describe 'check the content of the show page' do
```

Comment:
- [ ] Your post show page tests are missing the following scenarios.

  - Tests that asserts you can see how many comments it has.
  - Tests that asserts you can see how many likes it has.